source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -700,16 +702,97 @@ def apply_final_journey_filters_post_finalize(response_list, request):
         journey_pairs_pool = itertools.combinations(journeys, 2)
         _filter_similar_line_and_crowfly_journeys(journey_pairs_pool, request)
 
-    keep_olympics_journeys(response_list, request)
 
+def get_journey_pt_extremity(journey, clockwise):
+    if clockwise:
+        sections = reversed(journey.sections)
+    else:
+        sections = journey.sections
+    extemity_pt_section = next((s for s in sections if s.type == response_pb2.PUBLIC_TRANSPORT), None)
+
+    assert extemity_pt_section
+    if clockwise:
+        return extemity_pt_section.destination
+    return extemity_pt_section.arrival
```

Comment:
You use `.arrival` and `.destination` which seems weird (they're both talking about the end). Also, I don't see an `arrival` in the protobuf of a `Section`. Shouldn't this be `origin` and `destination`?